src/Wallabag/ApiBundle/Controller/WallabagRestController.php

```diff
@@ -122,11 +122,11 @@ public function postEntriesAction(Request $request)
             $this->get('wallabag_core.content_proxy')->assignTagsToEntry($entry, $tags);
         }
 
-        if (true === (bool) $isStarred) {
+        if ($isStarred === 'true') {
```

Comment:
Yep


> Le 16 mars 2016 à 19:29, Thomas Citharel <notifications@github.com> a écrit :
> 
> In src/Wallabag/ApiBundle/Controller/WallabagRestController.php:
> 
> > @@ -122,11 +122,11 @@ public function postEntriesAction(Request $request)
> >              $this->get('wallabag_core.content_proxy')->assignTagsToEntry($entry, $tags);
> >          }
> >  
> > -        if (true === (bool) $isStarred) {
> > +        if ($isStarred === 'true') {
> In that case, we should also change values retrieved to 0/1, what do you think ?
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
>